airflow/providers/amazon/aws/secrets/systems_manager.py

```diff
@@ -16,15 +16,13 @@
 # specific language governing permissions and limitations
 # under the License.
 """Objects relating to sourcing connections from AWS SSM Parameter Store"""
+import json
 from typing import Optional
 
 import boto3
 
-try:
-    from functools import cached_property
-except ImportError:
-    from cached_property import cached_property
-
+from airflow.compat.functools import cached_property
```

Comment:
The current try-except approach is intentional to keep the provider compatible with older Airflow versions. It shouldn't be changed unless you're going to bump the minimum Airflow requirement on this provider.